f2/server.js

```diff
@@ -167,10 +167,10 @@ app
     res.sendFile(path.join(__dirname, 'build', 'index.html'))
   })
 
-// uncomment to allow direct loading of arbitrary pages
-// .get('/*', function(_req, res) {
-//   res.sendFile(path.join(__dirname, 'build', 'index.html'))
-// })
+  // uncomment to allow direct loading of arbitrary pages
```

Comment:
Is this intentional for the staging server?

Reply:
whoops! good catch!

Reply:
good catch!